.gitignore

```diff
@@ -2,3 +2,4 @@ t/servroot*
 lua-resty-mlcache-*/
 *.tar.gz
 *.rock
+*.idea
```

Comment:
I'd recommend you add this to your global `~/.gitignore_global` file instead, and not to project-specific `.gitignore` files.